### PR TITLE
example: change detectron to yolov5

### PR DESCRIPTION
**Pull request by rickycao-qy**
Currently, we suggest faster rcnn based on detectron2 as default, this has several problems:
- Faster RCNN model is quite slow and has a high demand for hardware (GPU)
- Detectron2 framework needs to be compiled and built when installing, which may cause failures.

Thus, we can change the example object detecton2 to be yolov5, which is much faster.

This is the comparison between pipeline using yolov5 and detectron2(fater rcnn)
The pipeline is run on the [data](http://ai-sample.oss-cn-hangzhou.aliyuncs.com/image_classification/datasets/autoLayoutGroupRecognition.zip)
hardware: CPU Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz (5 cores)

| attr  | detectron2 faster rcnn   | yolov5 based on pytorch  |
|  ----  | ----  | ---- |
|  speed  | 15 days 13h  |4h 9min  |
| accuracy (map50) |  95.3%  | 99.5% |

Conclusion: we will suggest our users to use yolov5 pipeline both from speed and accuracy

### CODE REVIEW

File: docs/tutorials/component-object-detection.md [hunk 1]
```diff
@@ -167,29 +167,27 @@ After the dataset is ready, we can start training. Using Pipcook can be very con
...
-1. **@pipcook/plugins-detectron-model-train** This plugin is used to start the training of all kinds of models built on detectron2. We set iteration to 100000. If your dataset is very complex, you need to increase the number of iterations.
-1. **@pipcook/plugins-detectron-model-evaluate** We use this plug-in to evaluate the training effect of the model. This plugin will be effective only if the test testset is provided, and finally the average precision of each category is given.
-
-Since the object detection model, especially the model of the rcnn family is very large, it needs to be trained on a machine prepared with nvidia GPU and cuda 10.1 environment:
+1. **@pipcook/plugins-coco-data-access** Now that we have downloaded the dataset, we need to connect the dataset into the format required by the subsequent model. Since the model plugin used requires the coco format, we use this plugin.
+1. **@pipcook/plugins-pytorch-yolov5-model-define** We built a yolov5 model. This model has a very good performance in the accuracy of object detection
```
Comment (yorkie) [inline docs/tutorials/component-object-detection.md:188]:
```suggestion
2. **@pipcook/plugins-pytorch-yolov5-model-define** We built a yolov5 model. This model has a very good performance in the accuracy of object detection
```

File: docs/tutorials/component-object-detection.md [hunk 2]
```diff
@@ -167,29 +167,27 @@ After the dataset is ready, we can start training. Using Pipcook can be very con
...
+1. **@pipcook/plugins-pytorch-yolov5-model-train** This plugin is used to start the training of yolov5. We set epochs to 300. If your dataset is very complex, you need to increase the number of iterations.
```
Comment (yorkie) [inline docs/tutorials/component-object-detection.md:189]: Ditto

File: docs/tutorials/component-object-detection.md [hunk 3]
```diff
@@ -167,29 +167,27 @@ After the dataset is ready, we can start training. Using Pipcook can be very con
...
+1. **@pipcook/plugins-pytorch-yolov5-model-evaluate** We use this plug-in to evaluate the training effect of the model. This plugin will be effective only if the test testset is provided, and finally the average precision of each category is given.
```
Comment (yorkie) [inline docs/tutorials/component-object-detection.md:190]: Ditto

File: packages/cli/assets/example/pipelines/object-detection.json [hunk 4]
```diff
@@ -10,16 +10,16 @@
...
-      "package": "@pipcook/plugins-detectron-model-train",
+      "package": "@pipcook/plugins-pytorch-yolov5-model-train",
       "params": {
-        "steps": 100000
+        "epochs": 300
       }
     },
     "modelEvaluate": {
-      "package": "@pipcook/plugins-detectron-model-evaluate"
+      "package": "@pipcook/plugins-pytorch-yolov5-model-evaluate"
```
Comment (yorkie) [inline packages/cli/assets/example/pipelines/object-detection.json:22]: Shall we create a new pipeline that is still using detectron2?

**Comment by rickycao-qy** [comment]
We need to provide the baseline of performance between yolov5 and detectron2, from the perspective of both speed and accuracy. This will be done soon

File: docs/tutorials/component-object-detection.md (hunk 1 above)
Comment (rickycao-qy) [reply docs/tutorials/component-object-detection.md]: done

File: docs/tutorials/component-object-detection.md (hunk 3 above)
Comment (rickycao-qy) [reply docs/tutorials/component-object-detection.md]: done

**Comment by rickycao-qy** [comment]
@yorkie Please have a look again. The data of performance has been given. Also, give both pipelines examples of yolo and rcnn

File: example/pipelines/object-detection-fasterrcnn.json [hunk 5]
```diff
@@ -22,4 +22,4 @@
       "package": "@pipcook/plugins-detectron-model-evaluate"
     }
   }
-}
+}
```
Comment (yorkie) [inline example/pipelines/object-detection-fasterrcnn.json:25]: Unnecessary change?

**Comment by yorkie** [comment]
There are 2 small nits:

- https://github.com/alibaba/pipcook/pull/652#discussion_r539893741
- https://github.com/alibaba/pipcook/pull/652#discussion_r533922654

File: example/pipelines/object-detection-fasterrcnn.json (hunk 5 above)
Comment (rickycao-qy) [reply example/pipelines/object-detection-fasterrcnn.json]: done

File: packages/cli/assets/example/pipelines/object-detection.json (hunk 4 above)
Comment (rickycao-qy) [reply packages/cli/assets/example/pipelines/object-detection.json:22]: Yes, I remain the detectron2 pipeline and add the yolo pipeline

**Review by yorkie — APPROVED** [review]

**Review by WenheLI — APPROVED** [review]
LGTM
